sections/featured-collection.liquid

```diff
@@ -28,10 +28,19 @@
     assign products_to_display = section.settings.products_to_show
     assign more_in_collection = true
   endif
+
+  assign columns_mobile_int = section.settings.columns_mobile | plus: 0
+  assign show_mobile_slider = false
+  if section.settings.collection.all_products_count >= 2 and section.settings.swipe_on_mobile and section.settings.products_to_show >= 2
+    assign show_mobile_slider = true
+  endif
+  if columns_mobile_int == section.settings.products_to_show or columns_mobile_int == section.settings.collection.all_products_count
+    assign show_mobile_slider = false
+  endif
```

Comment:
Since we already have `product_to_display` we can probably leverage it to make the assign a little simpler. 
Here is what I have on my PR: 
```liquid
assign show_slider_on_mobile = false
  if section.settings.enable_mobile_slider and products_to_display > 2
    assign show_slider_on_mobile = true
  endif

  assign show_slider_on_desktop = false
  if section.settings.enable_desktop_slider and products_to_display > section.settings.columns_desktop
    assign show_slider_on_desktop = true
  endif
```

And if we keep your `columns_mobile_int` we could do: 
```liquid
assign columns_mobile_int = section.settings.columns_mobile | plus: 0
assign show_slider_on_mobile = false
  if section.settings.enable_mobile_slider and products_to_display > columns_mobile_int
    assign show_slider_on_mobile = true
  endif

  assign show_slider_on_desktop = false
  if section.settings.enable_desktop_slider and products_to_display > section.settings.columns_desktop
    assign show_slider_on_desktop = true
  endif
```

Reply:
I guess without the desktop stuff since that's added in my PR 😅